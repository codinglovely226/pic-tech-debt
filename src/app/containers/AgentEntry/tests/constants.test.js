import {
  ADD_AGENT,
  ADD_AGENT_CENTRE_AGENT,
  DELETE_AGENT,
  HIDE_AGENT_MODAL,
  REMOVE_AGENT_CENTRE_AGENT,
  REMOVE_AGENT_VIEW_ATTACHMENT,
  REMOVE_QUICK_VIEW_AGENT_NOTE_ATTACHMENT,
  RESET_AGENT_VIEW,
  RESET_QUICK_VIEW,
  SET_AGENT_TAB,
  SHOW_AGENT_MODAL,
  UPDATE_AGENT_CATEGORY,
  UPDATE_AGENT_CENTRE_AGENT,
  UPDATE_AGENT_DURATION,
  UPDATE_AGENT_EUPCS,
  UPDATE_AGENT_INTENDED_USE_GIVEN,
  UPDATE_AGENT_INTERLOCUTOR_AGENT_NAME,
  UPDATE_AGENT_IS_PATIENTS_OWN_MEDICATION,
  UPDATE_AGENT_QUANTITY,
  UPDATE_AGENT_ROUTE_OF_EXPOSURE,
  UPDATE_AGENT_TIME_OF_EXPOSURE,
  UPDATE_AGENT_TIME_SINCE_EXPOSURE,
  UPDATE_AGENT_TIME_SINCE_EXPOSURE_UNITS,
  UPDATE_AGENT_TYPE_OF_EXPOSURE,
  UPDATE_AGENT_USE_TYPE,
  UPDATE_AGENT_NOTES,
  UPDATE_AGENT_VIEW_CONTENT,
  UPDATE_AGENT_VIEW_ATTACHMENTS,
  UPDATE_QUICK_VIEW_AGENT_DURATION,
  UPDATE_QUICK_VIEW_AGENT_DURATION_UNITS,
  UPDATE_QUICK_VIEW_AGENT_DURATION_UNKNOWN,
  UPDATE_QUICK_VIEW_AGENT_NAME,
  UPDATE_QUICK_VIEW_AGENT_NOTE_ATTACHMENTS,
  UPDATE_QUICK_VIEW_AGENT_NOTE_CONTENT,
  UPDATE_QUICK_VIEW_AGENT_QUANTITY,
  UPDATE_QUICK_VIEW_AGENT_QUANTITY_UNITS,
  UPDATE_QUICK_VIEW_AGENT_QUANTITY_UNKNOWN,
  UPDATE_QUICK_VIEW_AGENT_ROUTE_OF_EXPOSURE,
  UPDATE_QUICK_VIEW_AGENT_TIME_OF_EXPOSURE,
  UPDATE_QUICK_VIEW_AGENT_TIME_SINCE_EXPOSURE,
  UPDATE_QUICK_VIEW_AGENT_TIME_SINCE_EXPOSURE_UNITS,
  UPDATE_QUICK_VIEW_AGENT_TYPE_OF_EXPOSURE,
} from '../constants';

describe('AgentEntry action constants', () => {
  it('ADD_AGENT', () => {
    expect(ADD_AGENT).toMatchSnapshot();
  });
  it('ADD_AGENT_CENTRE_AGENT', () => {
    expect(ADD_AGENT_CENTRE_AGENT).toMatchSnapshot();
  });
  it('DELETE_AGENT', () => {
    expect(DELETE_AGENT).toMatchSnapshot();
  });
  it('HIDE_AGENT_MODAL', () => {
    expect(HIDE_AGENT_MODAL).toMatchSnapshot();
  });
  it('REMOVE_AGENT_CENTRE_AGENT', () => {
    expect(REMOVE_AGENT_CENTRE_AGENT).toMatchSnapshot();
  });
  it('REMOVE_AGENT_VIEW_ATTACHMENT', () => {
    expect(REMOVE_AGENT_VIEW_ATTACHMENT).toMatchSnapshot();
  });
  it('REMOVE_QUICK_VIEW_AGENT_NOTE_ATTACHMENT', () => {
    expect(REMOVE_QUICK_VIEW_AGENT_NOTE_ATTACHMENT).toMatchSnapshot();
  });
  it('RESET_AGENT_VIEW', () => {
    expect(RESET_AGENT_VIEW).toMatchSnapshot();
  });
  it('RESET_QUICK_VIEW', () => {
    expect(RESET_QUICK_VIEW).toMatchSnapshot();
  });
  it('SET_AGENT_TAB', () => {
    expect(SET_AGENT_TAB).toMatchSnapshot();
  });
  it('SHOW_AGENT_MODAL', () => {
    expect(SHOW_AGENT_MODAL).toMatchSnapshot();
  });
  it('UPDATE_AGENT_CATEGORY', () => {
    expect(UPDATE_AGENT_CATEGORY).toMatchSnapshot();
  });
  it('UPDATE_AGENT_CENTRE_AGENT', () => {
    expect(UPDATE_AGENT_CENTRE_AGENT).toMatchSnapshot();
  });
  it('UPDATE_AGENT_DURATION', () => {
    expect(UPDATE_AGENT_DURATION).toMatchSnapshot();
  });
  it('UPDATE_AGENT_EUPCS', () => {
    expect(UPDATE_AGENT_EUPCS).toMatchSnapshot();
  });
  it('UPDATE_AGENT_INTENDED_USE_GIVEN', () => {
    expect(UPDATE_AGENT_INTENDED_USE_GIVEN).toMatchSnapshot();
  });
  it('UPDATE_AGENT_INTERLOCUTOR_AGENT_NAME', () => {
    expect(UPDATE_AGENT_INTERLOCUTOR_AGENT_NAME).toMatchSnapshot();
  });
  it('UPDATE_AGENT_IS_PATIENTS_OWN_MEDICATION', () => {
    expect(UPDATE_AGENT_IS_PATIENTS_OWN_MEDICATION).toMatchSnapshot();
  });
  it('UPDATE_AGENT_QUANTITY', () => {
    expect(UPDATE_AGENT_QUANTITY).toMatchSnapshot();
  });
  it('UPDATE_AGENT_ROUTE_OF_EXPOSURE', () => {
    expect(UPDATE_AGENT_ROUTE_OF_EXPOSURE).toMatchSnapshot();
  });
  it('UPDATE_AGENT_TIME_OF_EXPOSURE', () => {
    expect(UPDATE_AGENT_TIME_OF_EXPOSURE).toMatchSnapshot();
  });
  it('UPDATE_AGENT_TIME_SINCE_EXPOSURE', () => {
    expect(UPDATE_AGENT_TIME_SINCE_EXPOSURE).toMatchSnapshot();
  });
  it('UPDATE_AGENT_TIME_SINCE_EXPOSURE_UNITS', () => {
    expect(UPDATE_AGENT_TIME_SINCE_EXPOSURE_UNITS).toMatchSnapshot();
  });
  it('UPDATE_AGENT_TYPE_OF_EXPOSURE', () => {
    expect(UPDATE_AGENT_TYPE_OF_EXPOSURE).toMatchSnapshot();
  });
  it('UPDATE_AGENT_USE_TYPE', () => {
    expect(UPDATE_AGENT_USE_TYPE).toMatchSnapshot();
  });
  it('UPDATE_AGENT_NOTES', () => {
    expect(UPDATE_AGENT_NOTES).toMatchSnapshot();
  });
  it('UPDATE_AGENT_VIEW_CONTENT', () => {
    expect(UPDATE_AGENT_VIEW_CONTENT).toMatchSnapshot();
  });
  it('UPDATE_AGENT_VIEW_ATTACHMENTS', () => {
    expect(UPDATE_AGENT_VIEW_ATTACHMENTS).toMatchSnapshot();
  });
  it('UPDATE_QUICK_VIEW_AGENT_DURATION', () => {
    expect(UPDATE_QUICK_VIEW_AGENT_DURATION).toMatchSnapshot();
  });
  it('UPDATE_QUICK_VIEW_AGENT_DURATION_UNITS', () => {
    expect(UPDATE_QUICK_VIEW_AGENT_DURATION_UNITS).toMatchSnapshot();
  });
  it('UPDATE_QUICK_VIEW_AGENT_DURATION_UNKNOWN', () => {
    expect(UPDATE_QUICK_VIEW_AGENT_DURATION_UNKNOWN).toMatchSnapshot();
  });
  it('UPDATE_QUICK_VIEW_AGENT_NAME', () => {
    expect(UPDATE_QUICK_VIEW_AGENT_NAME).toMatchSnapshot();
  });
  it('UPDATE_QUICK_VIEW_AGENT_NOTE_ATTACHMENTS', () => {
    expect(UPDATE_QUICK_VIEW_AGENT_NOTE_ATTACHMENTS).toMatchSnapshot();
  });
  it('UPDATE_QUICK_VIEW_AGENT_NOTE_CONTENT', () => {
    expect(UPDATE_QUICK_VIEW_AGENT_NOTE_CONTENT).toMatchSnapshot();
  });
  it('UPDATE_QUICK_VIEW_AGENT_QUANTITY', () => {
    expect(UPDATE_QUICK_VIEW_AGENT_QUANTITY).toMatchSnapshot();
  });
  it('UPDATE_QUICK_VIEW_AGENT_QUANTITY_UNITS', () => {
    expect(UPDATE_QUICK_VIEW_AGENT_QUANTITY_UNITS).toMatchSnapshot();
  });
  it('UPDATE_QUICK_VIEW_AGENT_QUANTITY_UNKNOWN', () => {
    expect(UPDATE_QUICK_VIEW_AGENT_QUANTITY_UNKNOWN).toMatchSnapshot();
  });
  it('UPDATE_QUICK_VIEW_AGENT_ROUTE_OF_EXPOSURE', () => {
    expect(UPDATE_QUICK_VIEW_AGENT_ROUTE_OF_EXPOSURE).toMatchSnapshot();
  });
  it('UPDATE_QUICK_VIEW_AGENT_TIME_OF_EXPOSURE', () => {
    expect(UPDATE_QUICK_VIEW_AGENT_TIME_OF_EXPOSURE).toMatchSnapshot();
  });
  it('UPDATE_QUICK_VIEW_AGENT_TIME_SINCE_EXPOSURE', () => {
    expect(UPDATE_QUICK_VIEW_AGENT_TIME_SINCE_EXPOSURE).toMatchSnapshot();
  });
  it('UPDATE_QUICK_VIEW_AGENT_TIME_SINCE_EXPOSURE_UNITS', () => {
    expect(UPDATE_QUICK_VIEW_AGENT_TIME_SINCE_EXPOSURE_UNITS).toMatchSnapshot();
  });
  it('UPDATE_QUICK_VIEW_AGENT_TYPE_OF_EXPOSURE', () => {
    expect(UPDATE_QUICK_VIEW_AGENT_TYPE_OF_EXPOSURE).toMatchSnapshot();
  });
});
