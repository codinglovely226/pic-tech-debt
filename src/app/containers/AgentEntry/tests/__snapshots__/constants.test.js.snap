exports[`AgentEntry action constants ADD_AGENT 1`] = `"pic/AgentEntry/ADD_AGENT"`;

exports[`AgentEntry action constants ADD_AGENT_CENTRE_AGENT 1`] = `"pic/AgentEntry/ADD_AGENT_CENTRE_AGENT"`;

exports[`AgentEntry action constants DELETE_AGENT 1`] = `"pic/AgentEntry/DELETE_AGENT"`;

exports[`AgentEntry action constants HIDE_AGENT_MODAL 1`] = `"pic/AgentEntry/HIDE_AGENT_MODAL"`;

exports[`AgentEntry action constants REMOVE_AGENT_CENTRE_AGENT 1`] = `"pic/AgentEntry/REMOVE_AGENT_CENTRE_AGENT"`;

exports[`AgentEntry action constants REMOVE_AGENT_VIEW_ATTACHMENT 1`] = `undefined`;

exports[`AgentEntry action constants REMOVE_QUICK_VIEW_AGENT_NOTE_ATTACHMENT 1`] = `undefined`;

exports[`AgentEntry action constants RESET_AGENT_VIEW 1`] = `"pic/AgentEntry/RESET_AGENT_VIEW"`;

exports[`AgentEntry action constants RESET_QUICK_VIEW 1`] = `"pic/AgentEntry/RESET_QUICK_VIEW"`;

exports[`AgentEntry action constants SET_AGENT_TAB 1`] = `"pic/AgentEntry/SET_AGENT_TAB"`;

exports[`AgentEntry action constants SHOW_AGENT_MODAL 1`] = `"pic/AgentEntry/SHOW_AGENT_MODAL"`;

exports[`AgentEntry action constants UPDATE_AGENT_CATEGORY 1`] = `"pic/AgentEntry/UPDATE_AGENT_CATEGORY"`;

exports[`AgentEntry action constants UPDATE_AGENT_CENTRE_AGENT 1`] = `"pic/AgentEntry/UPDATE_AGENT_CENTRE_AGENT"`;

exports[`AgentEntry action constants UPDATE_AGENT_DURATION 1`] = `"pic/AgentEntry/UPDATE_AGENT_DURATION"`;

exports[`AgentEntry action constants UPDATE_AGENT_EUPCS 1`] = `"pic/AgentEntry/UPDATE_AGENT_EUPCS"`;

exports[`AgentEntry action constants UPDATE_AGENT_INTENDED_USE_GIVEN 1`] = `"pic/AgentEntry/UPDATE_AGENT_INTENDED_USE_GIVEN"`;

exports[`AgentEntry action constants UPDATE_AGENT_INTERLOCUTOR_AGENT_NAME 1`] = `"pic/AgentEntry/UPDATE_AGENT_INTERLOCUTOR_AGENT_NAME"`;

exports[`AgentEntry action constants UPDATE_AGENT_IS_PATIENTS_OWN_MEDICATION 1`] = `"pic/AgentEntry/UPDATE_AGENT_IS_PATIENTS_OWN_MEDICATION"`;

exports[`AgentEntry action constants UPDATE_AGENT_NOTES 1`] = `undefined`;

exports[`AgentEntry action constants UPDATE_AGENT_QUANTITY 1`] = `"pic/AgentEntry/UPDATE_AGENT_QUANTITY"`;

exports[`AgentEntry action constants UPDATE_AGENT_ROUTE_OF_EXPOSURE 1`] = `"pic/AgentEntry/UPDATE_AGENT_ROUTE_OF_EXPOSURE"`;

exports[`AgentEntry action constants UPDATE_AGENT_TIME_OF_EXPOSURE 1`] = `"pic/AgentEntry/UPDATE_AGENT_TIME_OF_EXPOSURE"`;

exports[`AgentEntry action constants UPDATE_AGENT_TIME_SINCE_EXPOSURE 1`] = `"pic/AgentEntry/UPDATE_AGENT_TIME_SINCE_EXPOSURE"`;

exports[`AgentEntry action constants UPDATE_AGENT_TIME_SINCE_EXPOSURE_UNITS 1`] = `"pic/AgentEntry/UPDATE_AGENT_TIME_SINCE_EXPOSURE_UNITS"`;

exports[`AgentEntry action constants UPDATE_AGENT_TYPE_OF_EXPOSURE 1`] = `"pic/AgentEntry/UPDATE_AGENT_TYPE_OF_EXPOSURE"`;

exports[`AgentEntry action constants UPDATE_AGENT_USE_TYPE 1`] = `"pic/AgentEntry/UPDATE_AGENT_USE_TYPE"`;

exports[`AgentEntry action constants UPDATE_AGENT_VIEW_ATTACHMENTS 1`] = `undefined`;

exports[`AgentEntry action constants UPDATE_AGENT_VIEW_CONTENT 1`] = `undefined`;

exports[`AgentEntry action constants UPDATE_QUICK_VIEW_AGENT_DURATION 1`] = `"pic/AgentEntry/UPDATE_QUICK_VIEW_AGENT_DURATION"`;

exports[`AgentEntry action constants UPDATE_QUICK_VIEW_AGENT_DURATION_UNITS 1`] = `"pic/AgentEntry/UPDATE_QUICK_VIEW_AGENT_DURATION_UNITS"`;

exports[`AgentEntry action constants UPDATE_QUICK_VIEW_AGENT_DURATION_UNKNOWN 1`] = `"pic/AgentEntry/UPDATE_QUICK_VIEW_AGENT_DURATION_UNKNOWN"`;

exports[`AgentEntry action constants UPDATE_QUICK_VIEW_AGENT_NAME 1`] = `"pic/AgentEntry/UPDATE_QUICK_VIEW_AGENT_NAME"`;

exports[`AgentEntry action constants UPDATE_QUICK_VIEW_AGENT_NOTE_ATTACHMENTS 1`] = `undefined`;

exports[`AgentEntry action constants UPDATE_QUICK_VIEW_AGENT_NOTE_CONTENT 1`] = `undefined`;

exports[`AgentEntry action constants UPDATE_QUICK_VIEW_AGENT_QUANTITY 1`] = `"pic/AgentEntry/UPDATE_QUICK_VIEW_AGENT_QUANTITY"`;

exports[`AgentEntry action constants UPDATE_QUICK_VIEW_AGENT_QUANTITY_UNITS 1`] = `"pic/AgentEntry/UPDATE_QUICK_VIEW_AGENT_QUANTITY_UNITS"`;

exports[`AgentEntry action constants UPDATE_QUICK_VIEW_AGENT_QUANTITY_UNKNOWN 1`] = `"pic/AgentEntry/UPDATE_QUICK_VIEW_AGENT_QUANTITY_UNKNOWN"`;

exports[`AgentEntry action constants UPDATE_QUICK_VIEW_AGENT_ROUTE_OF_EXPOSURE 1`] = `"pic/AgentEntry/UPDATE_AGENT_VIEW_ROUTE_OF_EXPOSURE"`;

exports[`AgentEntry action constants UPDATE_QUICK_VIEW_AGENT_TIME_OF_EXPOSURE 1`] = `"pic/AgentEntry/UPDATE_QUICK_VIEW_AGENT_TIME_OF_EXPOSURE"`;

exports[`AgentEntry action constants UPDATE_QUICK_VIEW_AGENT_TIME_SINCE_EXPOSURE 1`] = `"pic/AgentEntry/UPDATE_QUICK_VIEW_AGENT_TIME_SINCE_EXPOSURE"`;

exports[`AgentEntry action constants UPDATE_QUICK_VIEW_AGENT_TIME_SINCE_EXPOSURE_UNITS 1`] = `"pic/AgentEntry/UPDATE_QUICK_VIEW_AGENT_TIME_SINCE_EXPOSURE_UNITS"`;

exports[`AgentEntry action constants UPDATE_QUICK_VIEW_AGENT_TYPE_OF_EXPOSURE 1`] = `"pic/AgentEntry/UPDATE_QUICK_VIEW_AGENT_TYPE_OF_EXPOSURE"`;
